docs/developing-server.md

```diff
@@ -61,6 +61,8 @@ MAILER_USER="auth"
 MAILER_PASSWORD="auth"
 MAILER_HOST="localhost"
 MAILER_PORT="1025"
+STRIPE_API_KEY=sk_live_1
```

Comment:
add a comment "To debug Payment features"

Reply:
This was added to make sure the generated `schema.gql` will include payment related. Not sure if we should ignore schema.gql and related auto-generated files from git?

Reply:
that doesn't make sense as well, the schema generation in frontend requires `schema.gql`

Reply:
should be related to https://github.com/toeverything/AFFiNE/blob/282b788258ae6f3699cd82a751aeda182fd8cc3d/packages/backend/server/src/plugins/payment/index.ts#L21-L24

Without these two env var, when running `yarn dev` the schema.gql will regenerated and exclude the ones that defined in payment module.